### PR TITLE
Dev/adtang/acs phone numbers sms release nov2021

### PR DESCRIPTION
**Pull request by whisper6284**
Create new version of Communication SMS and Communication Phone Numbers packages.

Both versions are bumped from 1.0.5 to 1.0.6.

Both versions include the following dependency updates
- azure-core 1.22.0
- azure-communication-common -> 1.0.6
- azure-identity -> 1.4.1

### CODE REVIEW

File: sdk/communication/azure-communication-phonenumbers/CHANGELOG.md [hunk 1]
```diff
@@ -1,15 +1,14 @@
 # Release History
 
-## 1.1.0-beta.1 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
+## 1.0.6 (2021-11-09)
```
Comment (g2vinay) [inline sdk/communication/azure-communication-phonenumbers/CHANGELOG.md:3]: The date should be of today/tomorrow.
Depending on whenever you will release from release pipeline.

File: sdk/communication/azure-communication-sms/CHANGELOG.md [hunk 2]
```diff
@@ -1,15 +1,14 @@
 # Release History
 
-## 1.1.0-beta.1 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
+## 1.0.6 (2021-11-09)
```
Comment (g2vinay) [inline sdk/communication/azure-communication-sms/CHANGELOG.md:3]: Update date as above.

File: sdk/communication/azure-communication-phonenumbers/CHANGELOG.md (hunk 1 above)
Comment (whisper6284) [reply sdk/communication/azure-communication-phonenumbers/CHANGELOG.md]: Fixed.

File: sdk/communication/azure-communication-sms/CHANGELOG.md (hunk 2 above)
Comment (whisper6284) [reply sdk/communication/azure-communication-sms/CHANGELOG.md]: Fixed.

**Pull request merged**